Activity.js

```diff
@@ -9,10 +9,48 @@ class Activity {
   }
 
   startTimer() {
-
+      var currentTime = Date.parse(new Date());
+      var nye = new Date(currentTime + parseInt(minutesInput.value)*60*1000 + parseInt(secondsInput.value)*1000);
+      
+      var second = 1000;
+      var minute = second * 60;
+      var hour = minute * 60;
+      var day = hour * 24;
+      var timer;
+      
+      function determineRemaining() {
+        var today = new Date();
+        var distance = nye - today;
+        if (distance < 0) {
+          alert('Activity complete. What\'s next?')
+          clearInterval(timer);
+          return;
+        }
+        
+        var days = Math.floor(distance / day);
+        var hours = Math.floor((distance % day) / hour);
+        var minutes = Math.floor((distance % hour) / minute);
+        var seconds = Math.floor((distance % minute) / second);
+        
+        return {
+          days, hours, minutes, seconds
+        }
+      }
+      
+      function showRemaining() {
+        var remaining = determineRemaining();
+        document.querySelector('.countdown-minutes').innerHTML = formatNumber(remaining.minutes);
+        document.querySelector('.countdown-seconds').innerHTML = formatNumber(remaining.seconds);
+      }
+      
+      showRemaining();
```

Comment:
@m1073496  This is the key line to avoid the weird delay we were getting! We just needed to invoke `showRemaining` one time _before_ hopping into the `setInterval` function. This was described on the resource that Whitney posted in Slack, which is also on the Resources column of our project board.